grunttasks/lib/experiments.js

```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var EXPERIMENTS_FILE = './server/config/production-experiments.json';
```

Comment:
This `EXPERIMENTS_FILE` value is duplicated here and in the [./grunttasks/bump.js](https://github.com/mozilla/fxa-content-server/pull/2725/files#diff-becc724e0df9a64e398a42d62bc54ffdR14) file.
Not sure if we can normalize in one place and pass as a param or something.